[2025-02-17T11:11:42.708+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-17T11:11:42.749+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: group_task_dag.task_group.task8 scheduled__2025-02-15T21:33:00+00:00 [queued]>
[2025-02-17T11:11:42.784+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: group_task_dag.task_group.task8 scheduled__2025-02-15T21:33:00+00:00 [queued]>
[2025-02-17T11:11:42.784+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-17T11:11:42.804+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task_group.task8> on 2025-02-15 21:33:00+00:00
[2025-02-17T11:11:42.813+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6812) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-17T11:11:42.815+0000] {standard_task_runner.py:72} INFO - Started process 6818 to run task
[2025-02-17T11:11:42.818+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'group_task_dag', 'task_group.task8', 'scheduled__2025-02-15T21:33:00+00:00', '--job-id', '927', '--raw', '--subdir', 'DAGS_FOLDER/09_group_task_dag.py', '--cfg-path', '/tmp/tmpa579hed5']
[2025-02-17T11:11:42.820+0000] {standard_task_runner.py:105} INFO - Job 927: Subtask task_group.task8
[2025-02-17T11:11:42.904+0000] {task_command.py:467} INFO - Running <TaskInstance: group_task_dag.task_group.task8 scheduled__2025-02-15T21:33:00+00:00 [running]> on host ab74d4d90d32
[2025-02-17T11:11:43.141+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='group_task_dag' AIRFLOW_CTX_TASK_ID='task_group.task8' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T21:33:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-15T21:33:00+00:00'
[2025-02-17T11:11:43.142+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-17T11:11:43.170+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-17T11:11:43.171+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-17T11:11:43.188+0000] {subprocess.py:99} INFO - Output:
[2025-02-17T11:11:48.191+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-17T11:11:48.455+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-17T11:11:48.456+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=group_task_dag, task_id=task_group.task8, run_id=scheduled__2025-02-15T21:33:00+00:00, execution_date=20250215T213300, start_date=20250217T111142, end_date=20250217T111148
[2025-02-17T11:11:48.673+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-17T11:11:48.726+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-17T11:11:48.738+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
